Realm/Realm.Unity/Editor/Common/Analytics.cs

```diff
@@ -117,7 +117,7 @@ private string JsonPayload
                     .Replace("%SYNC_ENABLED%", _isSyncEnabled.ToString())
 
                     // Version of weaver is expected to match that of the library.
-                    .Replace("%REALM_VERSION%", "System.Reflection.Assembly.GetExecutingAssembly().GetName().Version.ToString()")
```

Comment:
Can I assume that in the past `"System.Reflection.Assembly.GetExecutingAssembly().GetName().Version.ToString()"` was evaluated at a later time? Otherwise how could it work if passed as a string? Just curiosity

Reply:
This is what the PR fixes - it shouldn't have been inside quotes 😅